### ISSUE TITLE
newbie question

### ISSUE THREAD

**Issue by MrAda** [baxterworks-build/buildroot-skeleton#1]
I saw postings on gamaral's channel from you regarding building a skeleton buildroot.  I have not been able to get anythign to work from buildroot and so maybe you can tell me what you are trying to accomplish here?  If I can build build root for my RPi3 that is fast and boots like gamaral's Pi2 version, I'd be very happy.

Chris

**Comment by ghost** [comment]
First you'd have to tell me what you've tried and what errors you're
getting.

I don't actually have a Raspberry Pi 3 set up with serial out due to the
issues with baud rate.

**Comment by MrAda** [comment]
Hi Adam,

I have just tried to build from source.  I have done the official RPi defined build using their official tar ball and I have cloned git repository and built.  None of which would work once I got it onto the sdcard.  I have only been lucky with raspian and archlinux.  I am trying to evaulate with OS is fastest and what gamaral had was exactly what I was looking for but he seems to have vanished as he is not talking lately.
